pkg/apis/events/v1alpha1/cloudauditlogssource_types.go

```diff
@@ -123,6 +123,11 @@ func (*CloudAuditLogsSource) ConditionSet() *apis.ConditionSet {
 	return &auditLogsSourceCondSet
 }
 
+// Methods for identifiable interface
+func (s *CloudAuditLogsSource) GetIdentity() string {
```

Comment:
and I think you can do also:
```
_ kngcpduck.Identifiable         = (*CloudAuditLogsSource)(nil)
```
so that we have a compile time check. See some of the first lines in this file where we do that for other interfaces. 
Can you also add this for other sources?